packages/realm/src/app-services/App.ts

```diff
@@ -233,14 +233,28 @@ export class App<FunctionsFactoryType = DefaultFunctionsFactory, CustomDataType
     return Object.fromEntries(this.internal.allUsers.map((user) => [user.identity, User.get(user, this)]));
   }
 
-  public switchUser(): unknown {
-    throw new Error("Not yet implemented");
+  /**
+   * Switches the current user to the one specified in {@link user}.
+   * The new user must be currently logged in, otherwise this method will throw an error.
+   * @param user The new current user.
+   */
+  public switchUser(user: AnyUser) {
+    this.internal.switchUser(user.internal);
   }
 
+  /**
+   * Removes a user and their local data from the device. If the user is logged in, they will be logged out in the process.
+   * @param user The user to remove.
+   */
   public async removeUser(user: AnyUser) {
     await this.internal.removeUser(user.internal);
   }
 
+  /**
+   * Deletes a user from the server. The user is also removed from the device together with their local data.
+   * If the user is logged in, they will be logged out in the process.
+   * @param user The user to delete.
+   */
   public async deleteUser(user: AnyUser) {
     await this.internal.deleteUser(user.internal);
```

Comment:
These doc comments will conflict with https://github.com/realm/realm-js/pull/5926

Reply:
Better get that merged soon so we can start improving the docs 😂